README.md

```diff
@@ -1,40 +1,28 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# La Suite
+
+This repository houses the code and assets for *La Suite* landing page.
+
+*La Suite* is currently under construction, and we anticipate ongoing updates and enhancements to our content.  Stay tuned!
 
 ## Getting Started
 
+This is a [Next.js](https://nextjs.org/) project. 
```

Comment:
Telling people to `yarn install` here could be great :)

Reply:
yeah I hesitated, I add it "de ce pas" :)